<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHG FROG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="btn_phase">
          <property name="text">
           <string>Retrieve Phase</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="dropdown">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Spec + Stage</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Camera + Stage</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_connect">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(138, 226, 52);</string>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btn_measure">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Measure</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_collect_spectrum">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Collect Spectrum</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btn_roi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>ROI</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QProgressBar" name="progress">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_load">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Load Measurement Data</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_subtract_background">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Subtract Background</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_exit">
   <property name="text">
    <string>Save settings and exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
